.github/workflows/enable-pack-release.yaml

```diff
@@ -0,0 +1,123 @@
+name: Enable Pack Release
+
+on:
+  # First day of the month at midnight.
+  schedule:
+    - cron:  "0 0 1 * *"
+  # manual rebuilds by TSC members
+  workflow_dispatch:
+
+env:
+  ci_branch: master
+  exchange_tools_branch: master
+  packs_org: StackStorm-Exchange
+  pack_repo_prefix: stackstorm
+
+# Only allow one workflow run per branch at a time.
+# When one is already running, new runs will be pending.
+# https://github.blog/changelog/2021-04-19-github-actions-limit-workflow-run-or-job-concurrency/
+concurrency:
+  group: enable-pack-release-${{ github.ref }}
+  # Do not use cancel-in-progress! Or we might create a scenario where all workflows get canceled before they complete.
+
+permissions:
+  actions: write # Need to enable Actions
+  contents: read # Need to clone repositories
```

Comment:
These permissions apply to the auto-generated github.token. But, that token cannot get write access to any other repo. So, I think we'll need some other token for this, and probably stored as a secret in github. I can help create that - let me know what kind of token and what permissions are needed and I'll set it up.

Reply:
> These permissions apply to the auto-generated github.token. But, that token cannot get write access to any other repo. So, I think we'll need some other token for this, and probably stored as a secret in github. I can help create that - let me know what kind of token and what permissions are needed and I'll set it up.

Hrm, I guess fine-grained PATs are likely out which only leaves "Classic PATs":

```shell
❯ gh repo list
HTTP 401: Personal access tokens with fine grained access do not support the GraphQL API (https://api.github.com/graphql)
```

This seems to be the bare minimum:

<img width="472" alt="image" src="https://user-images.githubusercontent.com/6192266/208204707-c4193712-2620-4573-91a9-76824cf17095.png">

```shell
❯ gh workflow list --repo mamercad/dotfiles
Lint Code Base  active  2404312
integration     active  32473778

❯ gh workflow disable 2404312 --repo mamercad/dotfiles
✓ Disabled Lint Code Base

❯ gh workflow enable 2404312 --repo mamercad/dotfiles
✓ Enabled Lint Code Base
```

Reply:
How's [this](https://github.com/StackStorm-Exchange/ci/pull/137/files#diff-fec4ffbf93caa5b3185269c68882ee80ed1d1bd95675ba488c9743c2f096c58cR15) look?